### PR TITLE
Standardized import numpy as np in Doc/Tutorial

### PR DESCRIPTION
**Pull request by Aum8**
- [x] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [ ] I have read the ``CONTRIBUTING.rst`` file, have run ``pre-commit``
locally, and understand that continuous integration checks will be used to
confirm the Biopython unit tests and style checks pass with these changes.

- [ ] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

Closes #...

### CODE REVIEW

File: Doc/Tutorial/chapter_align.tex [hunk 1]
```diff
@@ -3047,11 +3047,11 @@ \subsection{Creating an Array object}
       alphabet='ACGT')
 \end{minted}
 
-\verb+Array+ objects can thus be used as an array and as a dictionary. They can be converted to plain numpy arrays or plain dictionary objects:
+\verb+Array+ objects can thus be used as an array and as a dictionary. They can be converted to plain Numpy arrays or plain dictionary objects:
 
 %cont-doctest
 \begin{minted}{pycon}
->>> import numpy
+>>> import np
```
[the remark this answers is not in the document]
Comment (Aum8) [reply Doc/Tutorial/chapter_align.tex:3054]: Oh I missed that

File: Doc/Tutorial/chapter_align.tex [hunk 2]
```diff
@@ -3073,7 +3073,7 @@ \subsection{Creating an Array object}
 While the \verb+alphabet+ property of an \verb+Array+ is immutable, you can create a new \verb+Array+ object by selecting the letters you are interested in from the alphabet. For example,
 %cont-doctest
 \begin{minted}{pycon}
->>> a = Array("ABCD", dims=2, data=numpy.arange(16).reshape(4, 4))
+>>> a = Array("ABCD", dims=2, data= np.arange(16).reshape(4, 4))
```
[the remark this answers is not in the document]
Comment (Aum8) [reply Doc/Tutorial/chapter_align.tex:3076]: Will that cause a problem?

**Comment by Aum8** [comment]
> Closing in favour of #4255 - I was hoping you'd avoid duplicating effort here.

I am new to this, so I was trying my hands on it.
I really appreciate your guidance on this. Thank you!

File: Doc/Tutorial/chapter_align.tex (hunk 2 above)
[the remark this answers is not in the document]
Comment (peterjc) [reply Doc/Tutorial/chapter_align.tex:3076]: The code would still work, but it fails our automated style checks using flake8.

**Comment by pvalle6** [comment]
Quick Tip, when making commits, try to keep the file name in the commit

**Comment by pvalle6** [comment]
> I am new to this, so I was trying my hands on it. I really appreciate your guidance on this. Thank you!
Quick Tip, when making commits, try to keep the file name in the commit

**Pull request closed without merging**